lib/utils.js

```diff
@@ -33,7 +33,7 @@ const parseSubject = (s) => {
   const matches = s.match(R)
   // unmatched to generic "others" category
   if (!matches) {
-    return ['others', s]
+    return ['others', 'others', s]
```

Comment:
🤔 would this introduce it to just look like:

```md
## others

### others

* item
* item
...
```

why not tailor it return something like `['others', undefined, s]` and work at the formatter level to list it out like:

```md

## others

- item
- item
...
```

Reply:
it won't bc the way it was planned, it already skips `others` for tier 2 in `formatNotes`  as highlighted in the code below
https://github.com/EQWorks/release/blob/2721eba4680c299ddaaea6d8f7f293595d6d7d65/lib/utils.js#L94

Reply:
because sometimes regex match results will also have t2 as undefined (cases wehre the comit is `release - did this and that`) and to organize the `acc` from the reducer to have them all together to be compiled in the right level, I'm adding them under a `others` key, so that's why it already knows to skip this subcategory.